main.py

```diff
@@ -1,19 +1,35 @@
 import os
 import sys
 
+from dataclasses import dataclass
 from pathlib import Path
 import click
+import shutil
 import numpy as np
 import requests
 import base64
 import cv2
 
+@dataclass
```

Comment:
Note for the future: 

A size method returning the tuple (width, height) would be convenient here as we often use that